src/assets/translations.data.json

```diff
@@ -1,7 +1,11 @@
 {
   "en": {
+    "app-name": "Hubs",
+    "app-short-domain": "hub.link",
```

Comment:
fyi we have this in the cloudformation stack, so I was planning on feeding it in at build time